docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ghcr.io/puppeteer/puppeteer:17.0.0
+# LABEL org.opencontainers.image.authors="Mogztter#Guillaume Grossetie"
+LABEL version="17.0.0"
+LABEL description="This docker image contains \
+asciidoctor-web-pdf to generate websites and pdf from asciidoc(tor)."
+
+ENV PUPPETEER_SKIP_DOWNLOAD=true \
+    PUPPETEER_EXECUTABLE_PATH=/usr/bin/google-chrome
+
+USER root
```

Comment:
Would it be possible to use a non-root user? 

Puppeteer is using a user named `pptruser`:

https://github.com/puppeteer/puppeteer/blob/ae4c5ea0312d7ad192f7a31b8f4b1aa9478ea0ea/docker/Dockerfile#L24

I guess we could do something similar and use a user named `asciidoctor`.

Reply:
I think it it related to their troubleshooting, when the `pptruser` has insufficient privileges.

```bash
docker run -i --init \
 --cap-add=SYS_ADMIN \
--rm ghcr.io/puppeteer/puppeteer:latest \
 node -e "`cat docker/test/smoke-test.js`"`
```

I will test, if switching back to the pptruser has no side effects, so root is only used during the installation of the asciidoctor package.

```Dockerfile
# Run everything after as non-privileged user.
USER pptruser
```

Reply:
I added the `asciidoctor` user. It has implications on the docker run command, which I have added in the `README.md`.